### ISSUE TITLE
add to melpa

### ISSUE THREAD

**Issue by justinabrahms** [publicimageltd/delve#11]
I'd love to use this. Manually installing packages is a drag, though. Would you mind uploading this to melpa? Any reason not to?

**Comment by dmitrym0** [comment]
@justinabrahms  maybe something like this will be helpful?

```elisp
(use-package delve
  :straight (:repo "publicimageltd/delve"
             :host github
             :type git)
  :bind
  ;; the main entry point, offering a list of all stored collections
  ;; and of all open Delve buffers:
  (("<f12>" . delve))
  :config
  ;; set meaningful tag names for the dashboard query
  (setq delve-dashboard-tags '("tag1" "tag2"))
 ;; turn on delve-minor-mode when org roam file is opened:
  (delve-global-minor-mode))
```

**Comment by justinabrahms** [comment]
Thanks, @dmitrym0 but I don't actually have straight setup. It's okay if you don't close it, but this is an "I'd love to see this on melpa" request. :)

**Comment by publicimageltd** [comment]
thanks for asking! I don't feel that the package is ripe yet. There are some changes in the background waiting, and I wanted to get rid of that stupid restriction to a "storage directory" before I would publish it Because once it's out in the wild, all users would then have storage files in this directory with a wrong suffiix (it's `.el` now, it should be `.delve`), and with a clumsy workflow. My idea is to store the delve lists anywhere you want, e.g. in project directories. For that, I wanted to implement bookmarks and change the UX a bit. But I am stuck in non-elisp work now for quite some months, and I'm afraid it won't stop soon. 

So in short: Thanks for encouraging me, and yes, I will do that, but I want minimally to change the way Delve buffers are stored. Which might however take some time, because I have no time currently to dig even into these trivial things.

I'll leave that issue open, as a reminder.
